client.go

```diff
@@ -30,6 +30,7 @@ type Client struct {
 	project    string
 	userAgent  string
 
+	Emails          *EmailService
```

Comment:
I see where this pattern is coming from however I would like to suggest we call this `EmailDomains` instead. This service does not represent _emails_ but rather the _domains with email enabled/activated_ (amongst other things).

Even though there are technical details linking the `email/*` and `domain/*` parts of our API together they are not really the same _thing_ and we've been trying to differentiate them using the terms "DNS Domains" and "Email Domains".

Reply:
Thank you for the feedback, I agree that `EmailDomains` is a better name in the larger perspective. I'll push an update soon to change this. 👍